### PR TITLE
PIM-7067: test the UI to delete product models

### PR DESCRIPTION
**Pull request by docteurklein**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | OK
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | -
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: features/product-model/remove.feature [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Feature: Remove a product model
+  In order to delete an unnecessary product model from my PIM
+  As a product manager
+  I need to be able to remove a product model
+
+  Background:
+    Given a "catalog_modeling" catalog configuration
+    And I am logged in as "Julia"
+
+  @ce
```
Comment (damien-carcel) [inline features/product-model/remove.feature]: Do you have an incoming PR on EE for this scenario?

File: features/product-model/remove.feature [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  Scenario: Successfully delete a product model from the grid
+    Given I am on the products grid
+    When I click on the "Delete the product" action of the row which contains "amor"
+    Then I should see the text "Confirm deletion"
+    When I confirm the removal
+    Then I should not see product 1111111111
+    And  I should not see product 1111111112
```
Comment (damien-carcel) [inline features/product-model/remove.feature]: extra space

File: features/product-model/remove.feature (hunk 1 above)
Comment (docteurklein) [reply features/product-model/remove.feature]: yes, as discussed with @Delphineray, we keep it like that for the moment

Comment (Delphineray) [reply features/product-model/remove.feature]: yes, we will fix it in another card

File: features/product-model/remove.feature [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  In order to delete an unnecessary product model from my PIM
+  As a product manager
+  I need to be able to remove a product model
+
+  Background:
+    Given a "catalog_modeling" catalog configuration
+    And I am logged in as "Julia"
+
+  @ce
+  Scenario: Successfully delete a product model from the grid
```
Comment (aRn0D) [inline features/product-model/remove.feature]: When you remove a product model its children are deleted, is it the use case you describe here? If yes, please be more explicit.

File: features/product-model/remove.feature [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    Then I should see the text "Confirm deletion"
+    When I confirm the removal
+    Then I should not see product 1111111111
+    And  I should not see product 1111111112
+
+  Scenario: Successfully delete a product model from the edit form
+    Given I am on the "amor" product model page
+    And  I press the secondary action "Delete"
+    Then I should see the text "Confirm deletion"
+    When I confirm the removal
```
Comment (aRn0D) [inline features/product-model/remove.feature]: Are you redirected to the grid? if yes you should add a step and `I should be on the products page` or the next step should be `And the product 1111111111 is deleted`

**Pull request merged**